documentation/projects/proposals/popularity_optimizations/index.md

```diff
@@ -0,0 +1,5 @@
+# Popularity Optimizations
+
+```{include} /_toc_templates/flat.md
+
+```
```

Comment:
Note: this may need updating if #1868 is merged before this PR.

Reply:
Good to know, thank you!